content/docs/fairing/install-fairing.md

```diff
@@ -0,0 +1,240 @@
+Version: 0.5.2
+Summary: Python SDK for building, training, and deploying ML models
+Home-page: https://github.com/kubeflow/fairing
+Author: Kubeflow Authors
+Author-email: None
+License: UNKNOWN
+Location: <path-to-kubeflow-fairing>
+Requires: google-auth, six, setuptools, httplib2, cloudpickle, requests,
+future, kubernetes, google-cloud-storage, notebook, tornado, numpy,
+google-api-python-client, docker, oauth2client
+```
+
+### Docker setup
+
+You need to have Docker installed to use Kubeflow Fairing. Kubeflow Fairing
+packages your code as a docker image and executes it in a remote cluster.
+To check if your local Docker daemon is running, run the following command:
+
+```bash
+docker ps
+```
+
+*  If you get a message like `docker: command not found`, then [install
+   Docker](https://docs.docker.com/install/).
+*  If you get an error like `Error response from daemon: Bad response from
+   Docker engine`, then [restart your docker daemon][docker-start].
+*  If you are using Linux and you use sudo to access Docker, follow these
+   steps to [add your user to the docker group][docker-non-root]. Note, the
+   docker group grants privileges equivalent to the root user. To learn more
+   about how this affects security in your system, see the guide to the
+   [Docker daemon attack surface][docker-attack].
+
+### Configure Kubeflow Fairing
+
+To configure Kubeflow Fairing with access to the environment you would like to
+use for training and deployment, follow the instructions in the [guide to
+configuring Kubeflow Fairing][conf].
+
+## Set up Kubeflow Fairing in a hosted Jupyter notebook 
+
+Follow these instructions to set up Kubeflow Fairing in a hosted Jupyter
+notebook. 
+
+If you are using a Kubeflow notebook server that was built from one of the
+standard Jupyter Docker images, your notebooks environment has been
+preconfigured for training and deploying ML models with Kubeflow Fairing and
+no additional installation steps are required.  
+
+### Prerequisites
+
+Check the following prerequisites to verify that Kubeflow Fairing is compatible
+with your hosted notebook environment.
+
+1.  In the Jupyter notebooks user interface, click **File** > **New** >
+    **Terminal** in the menu to start a new terminal session in your notebook
+    environment.
+1.  You need **Python 3.6** or later to use Kubeflow Fairing. To check if you
+    have Python 3.6 or later installed, run the following command in your
+    terminal session:
+
+    ```bash
+    python3 -V
+    ```
+
+    The response should be something like this:
+
+    ```
+    Python 3.6.5
+    ```
+1.  You need to have Docker installed to use Kubeflow Fairing. Kubeflow Fairing
+    packages your code as a docker image and executes it in a remote cluster.
+    To check if your local Docker daemon is running, run the following command
+    in your terminal session:
+
+    ```bash
+    docker ps
+    ```
+
+    *  If you get a message like `docker: command not found`, then [install
```

Comment:
Apply some of the suggested edits from above.

Reply:
Done